### PR TITLE
Try to make travis matrice work again

### PR DESCRIPTION
**Pull request by FanchTheSystem**

### CODE REVIEW

**Comment by FanchTheSystem** [comment]
in fact ansible 2.10 should support python 2.7
but ansible 2.10.2 is not released yet : https://github.com/ansible-community/ansible-build-data/blob/main/2.10/CHANGELOG-v2.10.rst

File: .travis.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 sudo: required
-dist: xenial
```
Comment (tersmitten) [inline .travis.yml:3]: I prefer to stick to `xenial`, because that's what I'm running

**Comment by tersmitten** [comment]
That `latest` task in probably failing because the latest version of `ansible-lint` requires Python `3.6`...

**Comment by tersmitten** [comment]
See https://github.com/Oefenweb/ansible-postfix/issues/98

**Comment by FanchTheSystem** [comment]
> That `latest` task in probably failing because the latest version of `ansible-lint` requires Python `3.6`...

thank you ;)

**Comment by tersmitten** [comment]
Looks promising!

**Comment by FanchTheSystem** [comment]
All checks have passed :) 

Please merge or comment @tersmitten ;)

**Pull request merged**